chromadb/utils/batch_utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Tuple, List
+
+from chromadb.api.types import (
+    Documents,
+    Embeddings,
+    IDs,
+    Metadatas,
+)
+
+
+def create_batches(
```

Comment:
It'd be extra convenient from a client perspective to be able to pass here the API instead of the max_batch_size, and have the function extract the max_batch_size from the API object. That way the client doesn't need to call the API to get the max_batch_size to then pass it to this function. Saves one line of code ;) 

It is just a nice to have... maybe solvable by "overloading" this function.

Reply:
Done and tested.

Reply:
seems like this change wasn't made?

Reply:
I had made it but the rebase I did from main screw it up - https://github.com/amikos-tech/chroma-core/blob/c0d42f3dc8110371fc1a49d7db17af7e75cbbc8b/chromadb/utils/batch_utils.py

Reverting now.